synapse/models/syn.py

```diff
@@ -124,6 +160,51 @@ def _addRuntRows(self, form, valu, props, buidcache, propcache):
             if self.model.prop(prop).type.indx(propvalu):
                 propcache[(prop, propvalu)].append(buid)
 
+    async def _initCmdRunts(self):
+        now = s_common.now()
+        typeform = self.model.form('syn:cmd')
+        pkgmap = {}
+
+        for pkg in await self.core.getStormPkgs():
```

Comment:
Why does this data come from storm pkgs and not from data that is populated on the StormCmd class?  We could easily populate this and allow pure storm cmd implementers to benefit from this as well.